.github/dependabot.yml

```diff
@@ -4,13 +4,36 @@ updates:
   - package-ecosystem: cargo
     directory: /
     schedule:
-      interval: weekly
-      day: sunday
+      interval: daily
+      # UTC time
+      time: "06:00"
     rebase-strategy: disabled
+    commit-message:
+      prefix: "A0-3951: "
```

Comment:
Can we set these tasks as done in Jira? Otherwise, they will show up in our `old in-progress tasks` (and probably would have to be filtered out in Jira queries).

Reply:
Okay.